contracts/FIFOMarket.sol

```diff
@@ -154,27 +155,41 @@ contract FIFOMarket is
         suppliers[i] = supplier;
         remainingAmountToFill -= removalAmount;
       }
-
+      numberOfRemovals++;
```

Comment:
this is not actionable (not worth sacrificing readability), but leaving this here for reference 
![image](https://user-images.githubusercontent.com/18407013/165468432-f33ad7d9-7ba1-48ac-a206-c9bb507b6606.png)